### PR TITLE
Integrating PLC lib

### PR DESCRIPTION
**Pull request by dholms**
Integrating the new plc lib 🕺

### CODE REVIEW

File: packages/pds/service/index.js [hunk 1]
```diff
@@ -83,7 +86,8 @@ const smtpUrl = ({ username, password, host }) => {
 
 const getEnv = () => ({
   port: parseInt(process.env.PORT),
-  signingKeyId: process.env.SIGNING_KEY_ID,
+  repoSigningKey: process.env.REPO_SIGNING_KEY,
```
Comment (dholms) [inline packages/pds/service/index.js:89]: plz note that i changed these!

File: packages/pds/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "@atproto/pds",
   "version": "0.0.3",
   "license": "MIT",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```
Comment (devinivy) [inline packages/pds/package.json:5]:
```suggestion
  "main": "src/index.ts",
```

File: packages/pds/service/index.js [hunk 3]
```diff
@@ -36,7 +36,10 @@ const main = async () => {
   const cfInvalidator = new CloudfrontInvalidator({
     distributionId: env.cfDistributionId,
   })
-  const signingKeypair = await KmsKeypair.load({
+  const repoSigningKey = await KmsKeypair.load({
+    keyId: env.signingKeyId,
+  })
+  const plcRotationKey = await KmsKeypair.load({
     keyId: env.signingKeyId,
   })
```
Comment (devinivy) [inline packages/pds/service/index.js:44]: I think it's right, just confirming both are intended to be `env.signingKeyId` for good measure.

Comment (devinivy) [reply packages/pds/service/index.js:44]: Was able to clear this up myself by rereading the message you sent 😎

File: packages/pds/tests/_util.ts [hunk 4]
```diff
@@ -30,25 +31,48 @@ export const runTestServer = async (
...
+  }
+
+  // const plcDb =
+  //   dbPostgresUrl !== undefined
+  //     ? PlcDatabase.postgres({
+  //         url: dbPostgresUrl,
+  //         schema: `plc_test_${dbPostgresSchema}`,
+  //       })
+  //     : PlcDatabase.mock()
+  // await plcDb.migrateToLatestOrThrow()
```
Comment (devinivy) [inline packages/pds/tests/_util.ts:60]:
```suggestion
```

File: packages/pds/tests/handles.test.ts [hunk 5]
```diff
@@ -22,11 +23,12 @@ jest.mock('dns/promises', () => {
   }
 })
 
-describe.skip('handles', () => {
```
Comment (devinivy) [inline packages/pds/tests/handles.test.ts:25]: 🎉

**Review by devinivy — APPROVED** [review]
Good stuff!

**Review by devinivy — APPROVED** [review]

**Pull request merged**